portfolio/src/main/webapp/gallery.html

```diff
@@ -0,0 +1,24 @@
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">    
+    <title>Eline's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+    <div id="header">
+        <div id="logo">
+            <img width="200px" height="50" src="images/logo2.png" />
```

Comment:
SInce the navbar is reused in multiple files, might be good to move it to a separate file and include it here.